src/darwin/Framework/CHIP/templates/templates.json

```diff
@@ -91,6 +91,16 @@
             "path": "CHIPAttributeTLVValueDecoder-src.zapt",
             "name": "Decode TLV attribute values into Objc objects",
             "output": "src/darwin/Framework/CHIP/zap-generated/CHIPAttributeTLVValueDecoder.mm"
+        },
+        {
+            "path": "../../../../../examples/chip-tool-darwin/templates/CHIPTestClustersObjc.zapt",
```

Comment:
This part is wrong.  Update to tip, where zap_regen_all updates chip-tool-darwin correctly, please.

Reply:
- Was added as I thought it was needed, as discussed in this [slack conversationl](https://csamembers.slack.com/archives/G014G30SVV0/p1646935222327169)
- Removed in new push